api/batch/pom.xml

```diff
@@ -4,12 +4,12 @@
     <parent>
         <artifactId>rtd-ms-transaction-filter-api</artifactId>
         <groupId>it.gov.pagopa.rtd.ms.transaction_filter.api</groupId>
-        <version>1.2.0</version>
+        <version>1.2.1</version>
```

Comment:
@and-mora Maybe we could use a pom variable

Reply:
Partially done with version dependencies, thank you!